<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditorMenu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelSentenceConversation_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose Editor Type</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="leaderRadioButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Leader Editor</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="EventRadioButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Event Editor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openEditorButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Editor</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
